backend/census_historical_migration/admin.py

```diff
@@ -1,3 +1,11 @@
 from django.contrib import admin  # noqa: F401
 
-# Register your models here.
+from census_historical_migration.models import (
+    ELECAUDITHEADER,
+    ReportMigrationStatus,
+    MigrationErrorDetail,
+)
+
+admin.site.register(ELECAUDITHEADER)

```

Comment:
Let's add the remaining models then (`ELECEINS, ELECAUDITFINDINGS,..., ELEC***`). It's not necessary to include the test for all of them in this PR.